### PR TITLE
Added the logback project.

### PR DESCRIPTION
**Pull request by isomarcte**
* Fixed compiler warnings in the ScalaStyle test.
* Refactored into a multimodule build so downstream projects can only
  get the configs they want.
* Removed Apache 2 license and added Cotiviti License.
* Fixed bug in .gitignore for multimodule `target` folders.

### CODE REVIEW

**Comment by isomarcte** [comment]
Since this project is _currently_ open source, should I put the Apache 2 back for the time being?

**Comment by ctcreel** [comment]
Forgive my ignorance... what exactly is an open source project? This cotiviti-develtools-config?  If so then yes, it needs the Apache license.

**Comment by isomarcte** [comment]
Yes, `cotiviti-develtools-config` is currently public.

**Comment by isomarcte** [comment]
I'll switch the license and rebase one sec.

**Comment by isomarcte** [comment]
K updated.

**Comment by isomarcte** [comment]
Ugh, hold on. There is still a bad license in there...

**Comment by isomarcte** [comment]
Okay, it is good now.

**Comment by ctcreel** [comment]
ok... I had only one nit.  Fix that and I'll merge it.

**Comment by isomarcte** [comment]
@ctcreel I don't see your feedback anywhere.

**Comment by ctcreel** [comment]
you don't see my line level comments above?

**Comment by isomarcte** [comment]
nope

File: cotiviti-develtools-config-scalastyle/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+This repo provides a means of importing a standard ScalaStyle
+configuration and using it anywhere.
+
+It provides a XML resource file on the classpath at
+`io.cotiviti.cotiviti-scalastyle.xml`
+
+## I think these ScalaStyle rules are stupid ##
+
+Okay, maybe they are!
```
Comment (isomarcte) [inline cotiviti-develtools-config-scalastyle/README.md]: Do you want me to rephrase the section header? The "stuff below" is part of that section, but you asked me to pull out the header.

Comment (ctcreel) [inline cotiviti-develtools-config-scalastyle/README.md]: Yeah... the ScalaStyle rules aren't stupid.  They are a part of our development environment.  We don't have to justify that to anyone.

Comment (isomarcte) [inline cotiviti-develtools-config-scalastyle/README.md]: Done

**Pull request merged**